github_pr.py

```diff
@@ -104,7 +111,7 @@ def github_list_prs(**args):
     else:
         prs = repo.get_pulls()
         _print_prs(prs, **args)
-
+    return list_return_obj
```

Comment:
does this have a tab in front of it? the diff looks different than any other diff in this file (all preceding whitespace and the whole expression is highlighted)